CIP - ?/README.md

```diff
@@ -0,0 +1,120 @@
+---
+CIP: ?
```

Comment:
```suggestion
CIP: 83
```

Reply:
Removing number from PR title due to having been officially assigned to a proposal which was more active in this period: https://github.com/cardano-foundation/CIPs/blob/master/CIP-0083/README.md